### PR TITLE
Update mpConfig-2.0 to use released versions of MicroProfile and SmallRye Config

### PR DESCRIPTION
**Pull request by Joseph-Cass**
#build
#fat.buckets.to.run=io.openliberty.microprofile.config.2.0.internal_fat_tck,io.openliberty.microprofile.config.2.0.internal_fat

^^ This build command didn't work. See comment below.

### CODE REVIEW

**Comment by Joseph-Cass** [comment]
The above build command didn't do the job.

Hopefully this will work:

#build
#spawn.fullfat.buckets=io.openliberty.microprofile.config.2.0.internal_fat_tck,io.openliberty.microprofile.config.2.0.internal_fat

**Comment by Emily-Jiang** [comment]
@Joseph-Cass for some reason it did not compile judging from the Github Actions.

File: dev/cnf/oss_dependencies.maven [hunk 1]
```diff
@@ -120,8 +120,8 @@ io.smallrye.common:smallrye-common-classloader:1.5.0
 io.smallrye.common:smallrye-common-constraint:1.5.0
 io.smallrye.common:smallrye-common-expression:1.5.0
 io.smallrye.common:smallrye-common-function:1.5.0
-io.smallrye.config:smallrye-config-common:2.0.0-RC1
-io.smallrye.config:smallrye-config:2.0.0-RC1
+io.smallrye.config:smallrye-config-common:2.0.0-RC4
```
Comment (Emily-Jiang) [inline dev/cnf/oss_dependencies.maven:123]: The line is no longer correct. It should be
io.smallrye.config:smallrye-config-core:2.0.0-RC4

Comment (Emily-Jiang) [reply dev/cnf/oss_dependencies.maven:123]: please remove the lines 
io.smallrye.common:smallrye-common-annotation:1.5.0
io.smallrye.common:smallrye-common-classloader:1.5.0
io.smallrye.common:smallrye-common-constraint:1.5.0
io.smallrye.common:smallrye-common-expression:1.5.0
io.smallrye.common:smallrye-common-function:1.5.0

Comment (Emily-Jiang) [reply dev/cnf/oss_dependencies.maven:123]: after I talked with @Joseph-Cass , we do need this as they are the dependencies needed for resolving smallrye-config-core

**Comment by Joseph-Cass** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.config.2.0.internal_fat_tck,io.openliberty.microprofile.config.2.0.internal_fat

**Comment by Joseph-Cass** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.config.2.0.internal_fat_tck,io.openliberty.microprofile.config.2.0.internal_fat

**Comment by Joseph-Cass** [comment]
I'm just going to use this PR to update RC versions of MicroProfile Config and SmallRye Config to released versions

**Comment by Joseph-Cass** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.config.2.0.internal_fat_tck,io.openliberty.microprofile.config.2.0.internal_fat

**Review by Emily-Jiang — APPROVED** [review]
LGTM!

**Comment by Joseph-Cass** [comment]
#run-libby-bot

**Pull request merged**